Frameworks/AddressBook/ABAddressBookManagerInternal.h

```diff
@@ -17,10 +17,25 @@
 #pragma once
 
 #import "UWP/WindowsApplicationModelContacts.h"
+#import <AddressBook/ABRecord.h>
 
 @interface _ABAddressBookManager : NSObject
```

Comment:
The (roughly) equivalent Windows class is ContactManager (Also, one of the first code reviews for AddressBook mentioned that _ABAddressBook was a confusing name). Similarly, ABPerson's backing is called _ABContact (rather than _ABPerson) to match the Windows name of Contact

---
In reply to: [77094758](https://github.com/Microsoft/WinObjC/pull/935#discussion_r77094758) [](ancestors = 77094758)